src/main.rs

```diff
@@ -107,7 +107,7 @@ fn load_and_index_pois(mut rubber: Rubber, conn: &Connection, dataset: &str) {
             tags,
             'osm_poi_point' as source
             FROM osm_poi_point
-            WHERE name <> ''
+            WHERE name <> '' and ((mapping_key,subclass) not in ((highway,bus_stop), (barrier,gate), (amenity,waste_basket), (amenity,post_box), (tourism,information), (amenity,recycling), (barrier,lift_gate), (barrier,bollard), (barrier,cycle_barrier), (amenity,bicycle_rental), (tourism,artwork), (amenity,toilets), (leisure,playground), (amenity,telephone), (amenity,taxi), (leisure,pitch), (amenity,shelter), (barrier,sally_port), (barrier,stile), (amenity,ferry_terminal), (amenity,post_office)))
```

Comment:
I think it would be worth it to have it only once.

and since fafnir becomes more complex I think tests would be nice (and are now possible thanks to https://github.com/CanalTP/mimirsbrunn/pull/210)

Reply:
yup I agree.